db/fixtures/miq_user_roles.yml

```diff
@@ -442,6 +442,7 @@
   - vm_drift
   - vm_edit
   - vm_refresh
+  - vms_reload
```

Comment:
This is in the middle of `EvmRole-desktop`, but why not other ones?  (or do all of the other ones have a higher-level feature included?

Reply:
I wasn't sure, I just added it where vm_refresh was since the button is located in a similar place in the UI

Reply:
On further review I don't think we actually need these roles. Deleting them and updating the pr.